piqe_ocp_lib/api/resources/ocp_nodes.py

```diff
@@ -257,3 +258,82 @@ def execute_command_on_a_node(self, node_name, command_to_execute):
             "Command - %s - execution status:\n" "RETCODE: %s\nSTDOUT: %s\nSTDERR: %s\n", command, ret, out, err
         )
         return ret, out, err
+
+    def get_master_nodes(self) -> Optional["NodeList"]:
```

Comment:
Linting if failing with 
```
piqe_ocp_lib/api/resources/ocp_nodes.py:262:44: F821 undefined name 'NodeList'
piqe_ocp_lib/api/resources/ocp_nodes.py:275:44: F821 undefined name 'NodeList'
piqe_ocp_lib/api/resources/ocp_nodes.py:305:65: F821 undefined name 'Node'
piqe_ocp_lib/api/resources/ocp_nodes.py:319:67: F821 undefined name 'Node'
```
Note that although the 'kind' of resource you are dealing with is Node and NodeList,  the dynamic client abstracts all resources into RsourceInstance and ResourceList. For typing purposes you should be using ResourceInstance and ResourceList. The way you import those is as follows:
```
from openshift.dynamic.resource import ResourceInstance, ResourceList
```
To illustrate the difference
```
>>> from piqe_ocp_lib.api.resources import OcpNodes
>>> obj = OcpNodes()
>>> node_list = obj.get_all_nodes(label_selector="node-role.kubernetes.io/master")
>>> print(type(node_list))
<class 'openshift.dynamic.resource.ResourceInstance'>
>>> print(node_list.kind)
NodeList
```

Reply:
Thanks, that helps !!